command/launch.go

```diff
@@ -54,6 +56,8 @@ func (c *LaunchCommand) Run(args []string) int {
 	flags.StringVar(&userData, "userData", "", "User data")
 	flags.Int64Var(&volumeSize, "volumeSize", 8, "Volume size (default: 8)")
 	flags.StringVar(&volumeType, "volumeType", "gp2", "Volume type (default: gp2)")
+	flags.StringVar(&arn, "arn", "", "ARN")
```

Comment:
Sorry for my insufficient explanation...
I'd like to ask you: what RESOURCE is this ARN attached to?

ex. IAM Policy ARN, IAM Instance Profile ARN